### PR TITLE
Digital PUL?

### LINKED ISSUE #223: Update home page boilerplate
To what, I'm not sure, but we probably can't brand at Plum externally.

### PR DESCRIPTION
**Pull request by jpstroop**
Closes #223. I ran "Digital PUL" by @marvinmarvinb yesterday. He wasn't completely sold, but I think it's a better brand than Plum, so let's run with it for now.

Feedback welcome on the blurb.

### CODE REVIEW

File: app/views/welcome/index.html.erb [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-  <%=t('curation_concerns.institution.name') %> community to share its research and scholarly
-  work with a worldwide audience. Faculty, staff, and students can use
-  <%=t('curation_concerns.product_name') %> to collect their work in one location and create a
-  durable and citeable record of their papers, presentations, publications, data
-  sets, or other scholarly creations.
+  <%=t('curation_concerns.product_name') %> is a secure application that enables
+  <%=t('curation_concerns.institution.name') %> to efficiently carry out varied
+  and complex workflows related to digitization while adhereing to standards
+  and best practices which ensure that our content will be preserved and
+  accessbile to our community and the world for generations.
```
Comment (tpendragon) [inline app/views/welcome/index.html.erb]: accessible

**Comment by tpendragon** [comment]
Tests really want it to be called Plum.

**Comment by jpstroop** [comment]
fFxed (I think) and rebased. Didn't occur to me that that might have coverage.

**Comment by escowles** [comment]
:+1: (though I'm still hoping to come up with a backronym for POND)

**Comment by jpstroop** [comment]
Like Amelia Pond? Challenge on!

:tardis:

**Comment by escowles** [comment]
Well, I was thinking of upgrading the PUDL, but whatever floats your :boat:

**Comment by jpstroop** [comment]
Princeton ON Deposit

**Comment by escowles** [comment]
Princeton ONline Digitallibrary was the best I've come up with (though maybe that's a name for Pom, not Plum?)

**Comment by jpstroop** [comment]
or Display, but I don't really like display as our motivation...

**Comment by escowles** [comment]
Princeton ONline Discovery?

**Comment by jpstroop** [comment]
Yeah... not sure about that either. Plum and pom could probably share s
brand.

-Js

Sent via mobile. Please excuse typos, brevity, etc.

**Comment by escowles** [comment]
Though you know this is going to lead to trying to make a backronym for POCEAN for the next generation...

**Comment by jpstroop** [comment]
Backronym generator that changes the name every day?

**Comment by escowles** [comment]
Every day? Config file of backronyms with a random one selected per-request!

**Comment by jpstroop** [comment]
Oh, I was thinking a file of words starting with P O N & D

-Js

Sent via mobile. Please excuse typos, brevity, etc.

**Pull request merged**